compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -142,9 +143,14 @@ class Context(val contextId: Context.ContextId, languageVersion: LanguageVersion
     val compiledPackages = PureCompiledPackages(allSignatures, defns, compilerConfig)
     for {
       defn <- defns.get(LfDefRef(identifier))
-    } yield Speedy.Machine.fromScenarioSExpr(
-      compiledPackages,
-      defn.body,
+    } yield
+    // TODO: https://github.com/digital-asset/daml/issues/12208
+    //  plug the logging context properly in the scenario service
```

Comment:
We don’t actually see log statements in the scenario service so I think this isn’t all that crucial.